internal/aws.go

```diff
@@ -125,10 +131,15 @@ func assumeRoleFromAWS(arn, externalID string, request *Request) (*sts.AssumeRol
 	}
 
 	request.setLabel("aws.cache.assume_role", "miss")
-	request.log.Infof("Requesting STS Assume Role info for %s from AWS", arn)
-	req := stsService.AssumeRoleRequest(constructAssumeRoleInput(arn, externalID))
+	request.log.Infof("Requesting STS Assume Role info for %s from AWS with expiration of %s seconds", arn, roleDurationSeconds)
+	req := stsService.AssumeRoleRequest(constructAssumeRoleInput(arn, externalID, intRoleDurationSeconds))
+
+	credentialRequestTime := time.Now()
+	request.log.Infof("Credential Request time: %s", credentialRequestTime.String())
```

Comment:
I think this should be removed or moved to `Debug` level, same with the other message below